src/components/AppNavigationSettings/AppNavigationSettings.vue

```diff
@@ -81,8 +81,9 @@ export default {
 
 .slide-up-leave-active,
 .slide-up-enter-active {
-	transition-duration: var(--animation-quick);
+	transition-duration: 200ms;
```

Comment:
@jancborchardt we should probably have this in a variable somewhere?

Reply:
Yes, let's do it via variable. :) @korelstar could you adjust that?

Reply:
Yes, of course. I assume that we should put that variable to nextcloud/server (?). Should it be really "slow" or maybe "medium"?

Reply:
medium, and put slow at 300? :thinking:

Reply:
300ms is already in use in https://github.com/nextcloud/server/blob/f2e9babaabeaecdf84bc7ae57cc7be6c7e16e431/apps/files/css/files.scss, so this sound reasonable.

Reply:
... and many more: https://github.com/search?q=org%3Anextcloud+300ms&type=Code

Reply:
Sounds good! For simplicity ot might even be better to only have 2 animation speeds?

Reply:
@jancborchardt Yes, but then we have to decide which one to chose. There are also several usages of 200ms: https://github.com/search?q=org%3Anextcloud+200ms&type=Code
For the settings, I think 300ms is also fine. But I don't know if it is for the others.

Reply:
Let's go for 300ms and don't create a medium value :)

Reply:
https://github.com/nextcloud/server/pull/15107#issuecomment-483180903